Dockerfile

```diff
@@ -39,13 +39,11 @@ ENV PLOTLY_CONNECTOR_DATA_FOLDER="/plotly-connector"
 # Log to stdout in addition to the file
 ENV PLOTLY_CONNECTOR_LOG_TO_STDOUT="true"
 
+COPY . /usr/src/app
+
 # Install app dependencies
-COPY package.json /usr/src/app
-COPY yarn.lock /usr/src/app
 RUN yarn install
-RUN yarn run rebuild:modules:electron
```

Comment:
So, the reason that we run each of these `COPY` commands is so that the output of the `yarn install` command is cached and not reused if `yarn.lock` doesn't change. For reference, see:
- http://blog.nchaulet.fr/cache-npm-install-on-docker/
- https://stackoverflow.com/a/35774741/4142536

Reply:
Oh I see.. that's new information.. 😸